### ISSUE TITLE
Add support for emoji icons

### ISSUE THREAD

**Issue by tbeu** [impact/search#10]
Search for "xml" in the search field. The only hit contains an emoji Icon in ist description which is displayed as text.

**Comment by xogeny** [comment]
This is because the person who added the description for that library chose to include GitHub specific emoji markup in the description.  I'm rendering exactly the description the library author included.

An argument could be made that it would be nice if the emoji images were embedded in the impact search results.  I would accept a pull request for this.  But I couldn't find an existing package that takes HTML with emoji markup and returns HTML with emoji images embedded and I don't want to implement that functionality right now.

**Comment by tbeu** [comment]
Isn't it here https://github.com/node-modules/emoji?

**Comment by xogeny** [comment]
Perhaps.  I'll investigate.

**Comment by xogeny** [comment]
No.  The one you referenced converts unicode characters into the associated image.

The module required in this case must transform a string that uses **markup** into HTML.  [This one](http://hassankhan.me/emojify.js/) handles that.

**Comment by xogeny** [comment]
BTW, this is a very nice reference for the various markup patterns:

http://www.emoji-cheat-sheet.com/

**Comment by xogeny** [comment]
It should be noted that `emojify` seems to be the standard.  But it also includes a very large CSS file.  An alternative approach, [used here](https://github.com/frissdiegurke/emoji-parser), is to include a bunch of static images.  This means less downloading up front.

I don't really want to host all those static images within the GitHub page.  I think it is "neater" to just have the CSS fine.  So I'm more inclined to go with `emojify`.  But this is pretty low on my list of priorities.

**Comment by tbeu** [comment]
> BTW, this is a very nice reference for the various markup patterns:
http://www.emoji-cheat-sheet.com/

Guess what I use all the time? :smirk:

**Comment by dietmarw** [comment]
@tbeu Until this is resolved you might consider removing the emoji from your description.

**Comment by tbeu** [comment]
But they are really nice eye-catchers in the GitHub description. :jack_o_lantern:

**Comment by tbeu** [comment]
HNY!  :fireworks:

**Comment by tbeu** [comment]
:new: January :snowman:
